test/repository_test.rb

```diff
@@ -82,6 +82,18 @@
             assert updated_user.updated_at > @updated_at
           end
         end
+
+        describe 'when entity is not respond_to attribute' do
```

Comment:
add another test after the test on the line 52 for the scenario in which `created_at` accessor is not available, we should test for the fact that `created_at` is not going to be touched